embrace-android-otel-java/src/main/kotlin/io/embrace/android/embracesdk/otel/java/EmbraceOtelJavaExtensions.kt

```diff
@@ -0,0 +1,51 @@
+package io.embrace.android.embracesdk.otel.java
+
+import io.embrace.android.embracesdk.Embrace
+import io.embrace.opentelemetry.kotlin.ExperimentalApi
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaAttributeKey
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaLogRecordExporter
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaOpenTelemetry
+import io.embrace.opentelemetry.kotlin.aliases.OtelJavaSpanExporter
+import io.embrace.opentelemetry.kotlin.logging.export.toOtelKotlinLogRecordExporter
+import io.embrace.opentelemetry.kotlin.toOtelJavaApi
+import io.embrace.opentelemetry.kotlin.tracing.export.toOtelKotlinSpanExporter
+
+/**
+ * Adds an [OtelJavaSpanExporter] that OTel Spans will be exported to after completion.
+ */
+@OptIn(ExperimentalApi::class)
+fun Embrace.addJavaSpanExporter(spanExporter: OtelJavaSpanExporter) {
+    this.addSpanExporter(spanExporter.toOtelKotlinSpanExporter())
+}
+
+/**
+ * Adds an [OtelJavaLogRecordExporter] that OTel LogRecords will be exported to after completion.
+ */
+@OptIn(ExperimentalApi::class)
+fun Embrace.addJavaLogRecordExporter(logRecordExporter: OtelJavaLogRecordExporter) {
+    this.addLogRecordExporter(logRecordExporter.toOtelKotlinLogRecordExporter())
+}
+
+/**
+ * Returns an [OtelJavaOpenTelemetry] that provides working [Tracer] implementations that will record spans that fit into the Embrace data
+ * model.
+ */
+@OptIn(ExperimentalApi::class)
+fun Embrace.getJavaOpenTelemetry(): OtelJavaOpenTelemetry {
+    // TODO: Should we return noop if getOpenTelemetryKotlin is noop?
+    return this.getOpenTelemetryKotlin().toOtelJavaApi()
+}
+
+/**
+ * Set an attribute on the resource used by the OTel SDK instance with the given key and String value.
+ * The value set will override any value set previously or by the Embrace SDK.
+ * This must be called before the SDK is started in order for it to take effect.
+ */
+@Deprecated(
+    "Use setResourceAttribute(key: String, value: String) instead.",
+    ReplaceWith("setResourceAttribute(key.key, value)")
+)
+fun Embrace.setJavaResourceAttribute(
```

Comment:
This was called `setResourceAttribute` previously - why has the symbol name changed?

Reply:
To avoid having the same name for this and the `setResourceAttribute(key: String, value: String)` method. Same with `getJavaOpenTelemetry` and `addJavaSpanExporter`. Would you rather keep the names?

Reply:
IMO yes, but that depends on what we call the Kotlin variants of these functions and where they live. Perhaps we can chat about it internally?